pkg/cvo/cvo.go

```diff
@@ -652,3 +661,28 @@ func (optr *Operator) defaultPreconditionChecks() precondition.List {
 		preconditioncv.NewUpgradeable(optr.cvLister),
 	}
 }
+
+func (optr *Operator) getHTTPClientTransport() (http.RoundTripper, error) {
```

Comment:
should all https clients in cvo pkg use this? because the name most definitely implies this.